config/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/resource/OAuth2ResourceServerConfigurer.java

```diff
@@ -231,13 +242,20 @@ private void registerDefaultCsrfOverride(H http) {
 		csrf.ignoringRequestMatchers(this.requestMatcher);
 	}
 
-	private BearerTokenResolver getBearerTokenResolver() {
+	BearerTokenResolver getBearerTokenResolver() {
```

Comment:
This needs to be default so that unit tests can access it. What this comes down to is how we prefer to do unit tests for configurers. In the PR for JwtDecoder, the same tradeoff was made in order to allow for corner cases to be confirmed (as unit tests) instead of as performing a full MockMvc roundtrip.

Reply:
What's the disadvantage of doing MockMvc roundtrip? I see `SecurityConfigurers` more like integration testing instead of unit testing.

> What this comes down to is how we prefer to do unit tests for configurers

Can you clarify what you mean by "we prefer"? Did I miss a convo you and @rwinch had?